shell/pom.xml

```diff
@@ -69,5 +69,45 @@
       <artifactId>alluxio-core-common</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
```

Comment:
Discussed offline with @apc999 @aaudiber @madanadit. We decided to move validateEnv and related classes to the server-common module.